src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: YouTube enhances UX with Media Capabilities Web API
+subhead: Learn how Media Capabilities helped YouTube.
```

Comment:
I noticed a lot of subheads start with "Learn how..." which looks a bit repetitive on the blog. I was wondering if we could try something that grabs the reader a bit more like: "How do you ensure buttery smooth playback when all your users are on different devices?" That's just a suggestion, feel free to go with something else.

I've found it's helpful to look at the subheads on https://medium.com/ for inspiration.

Reply:
It's definitely better. Thanks for the suggestion!